documentation/functionalDoc/functionalDoc.tex

```diff
@@ -231,6 +231,24 @@ \subsubsection{Frequency models}
 
 This model is connected to all the generators of one synchronous area and takes their speed as inputs to calculate the reference speed that is used into the machine models.
 
+\subsubsection{Utilities}
+\label{Utilities}
+
+One model is implemented for computing various aggregations of voltage levels: \verb VoltageMeasurementUtilities  model.
```

Comment:
buses voltages instead of voltage levels. For example bus 1 with U = 401kV and bus 2 with U = 403kV are two buses of the 400kV voltage level